README.md

```diff
@@ -6,148 +6,108 @@
-* [Gate.io](https://www.gate.io/signup/5868285)
-* [AscendEx](https://ascendex.com/register?inviteCode=UEIXNXKW)
-* [Huobi](https://www.htx.com/)
-* [OKX](https://www.okx.com/join/1931920)
-
-For more information about the support provided by these partners, see the financial reports provided in [HBOT Tracker](https://docs.google.com/spreadsheets/d/1UNAumPMnXfsghAAXrfKkPGRH9QlC8k7Cu1FGQVL1t0M/edit#gid=285483484).
+* [Reported Volumes](https://p.datadoghq.com/sb/a96a744f5-a15479d77992ccba0d23aecfd4c87a52): Reported trading volumes across all Hummingbot instances
+* [Newsletter](https://hummingbot.substack.com): Get our newsletter whenever we ship a new release
+
+
+
+## Supported Exchange Connectors
+
+Hummingbot connectors standardize trading logic and order types across different types of exchanges:
+
+ * **CLOB Spot**: Connectors to spot markets on central limit order book (CLOB) exchanges
+ * **CLOB Perp**: Connectors to perpetual futures markets on CLOB exchanges
+ * **AMM**: Connectors to spot markets on Automatic Market Maker (AMM) decentralized exchanges
+
+### Partners and Sponsors
+
+We are grateful for the following exchange partners who support the development and maintenance of Hummingbot.
+
+| Connector ID | Exchange | Type | Discount |
+|----|------|-------|----------|
+| `binance` | [Binance](https://accounts.binance.com/register?ref=CBWO4LU6) | CLOB Spot | [![Sign up for Binance using Hummingbot's referral link for a 10% discount!](https://img.shields.io/static/v1?label=Fee&message=%2d10%25&color=orange)](https://accounts.binance.com/register?ref=CBWO4LU6) |
+| `binance_perpetual` | [Binance](https://accounts.binance.com/register?ref=CBWO4LU6) | CLOB Perp | [![Sign up for Binance using Hummingbot's referral link for a 10% discount!](https://img.shields.io/static/v1?label=Fee&message=%2d10%25&color=orange)](https://accounts.binance.com/register?ref=CBWO4LU6) |
+| `dydx_v4_perpetual` | [dYdX](https://www.dydx.exchange/) | CLOB Perp | - |
+| `hyperliquid_perpetual` | [Hyperliquid](https://hyperliquid.io/) | CLOB Perp | - |
+| `gate_io` | [Gate.io](https://www.gate.io/referral/invite/HBOTGATE_0_103) | CLOB Spot | [![Sign up for Gate.io using Hummingbot's referral link for a 10% discount!](https://img.shields.io/static/v1?label=Fee&message=%2d20%25&color=orange)](https://www.gate.io/referral/invite/HBOTGATE_0_103) |
+| `gate_io_perpetual` | [Gate.io](https://www.gate.io/referral/invite/HBOTGATE_0_103) | CLOB Perp | [![Sign up for Gate.io using Hummingbot's referral link for a 20% discount!](https://img.shields.io/static/v1?label=Fee&message=%2d20%25&color=orange)](https://www.gate.io/referral/invite/HBOTGATE_0_103) |
+| `htx` | [HTX (Huobi)](https://www.htx.com.pk/invite/en-us/1h?invite_code=re4w9223) | CLOB Spot | [![Sign up for HTX using Hummingbot's referral link for a 20% discount!](https://img.shields.io/static/v1?label=Fee&message=%2d20%25&color=orange)](https://www.htx.com.pk/invite/en-us/1h?invite_code=re4w9223) |
```

Comment:
here to add referral link
https://hummingbot.org/exchanges/huobi/#connector-info

Reply:
https://github.com/hummingbot/hummingbot-site/pull/405